Flint/src/test/java/unit/FlintDriverTest.java

```diff
@@ -0,0 +1,68 @@
+package unit;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.RandomAccessFile;
+
+import com.github.javaparser.ast.CompilationUnit;
+
+import org.junit.Test;
+
+import flint.FlintDriver;
+import mocks.TestConfig;
+
+public class FlintDriverTest {
+  @Test(expected = FileNotFoundException.class)
+  public void throwsExceptionForNonexistentFile() throws Exception {
+    FlintDriver.run("invalid-file.java", new TestConfig());
+  }
+
+  @Test
+  public void callsRunChecks() throws Exception {
+    TestConfig tc = new TestConfig();
+    FlintDriver.run("src/main/java/flint/FlintDriver.java", tc);
+    assertEquals(tc.getCallCount(), 1);
+  }
+
+  @Test
+  public void callsRunChecksWithRandomAccessFile() throws Exception {
```

Comment:
Just wondering why are you testing this? Since the doc didn't say that you are passing a `RandomAccessFile` to the `runCheck()` of config.

Reply:
Well, I think I figure this out. I think you are just testing if the code is passing the right thing.